Base/usr/share/man/man7/kernel_boot_parameters.md

```diff
@@ -0,0 +1,142 @@
+## Name
+
+Kernel Boot Parameters - strings recognized by the kernel, that determine the
+functionality in runtime.
+
+## Description
+
+The kernel supports boot parameters to change some of its behavior. This
+functionality can be used to disable or enable features in the OS entirely, but
+can sometimes be used also to change a feature's behavior without disabling it
+entirely.
+
+Some of the boot parameters require a value after it. In that case, you will
+need to write them like this:
+
+```
+
```

Comment:
You're adding way to much empty lines in this blocks. You're supposed to do it line this:

<pre><code>```
param1=value
```
</code></pre>

not like this:

<pre><code>```

param1=value

```
</code></pre>